cmake/options.cmake

```diff
@@ -159,6 +159,14 @@ set(ONEDNN_ENABLE_GEMM_KERNELS_ISA "ALL" CACHE STRING
       SSE41 < AVX2 < AVX512 < AMX (or ALL). It means that if user selects, e.g.
       AVX2 ISA, SSE41 kernels will also present at build time.")
 
+set(DNNL_HIP_TARGET_ARCH "" CACHE STRING
+    "Specifies the target offload architecture. Required when compiling
+    sycl kernels for AMD target backend.")
+if("${DNNL_HIP_TARGET_ARCH}" STREQUAL "")
+    add_definitions(-DDNNL_HIP_DISABLE_SYCL_KERNELS=1)
```

Comment:
Is there a reason why you use `disable` instead of `enable` here? It's a bit confusing because the SYCL kernels are disabled by default so I would expect that we use something like `add_definitions(-DNNL_HIP_ENABLE_SYCL_KERNELS)` to enable them.

Reply:
No particular reason but I agree enable makes more sense, changed it.